### DISCUSSION TITLE
Does Meta-AconC need much more GPU memory?

### DISCUSSION THREAD

**Discussion by zlj-ky** [ultralytics/yolov5#7422] [Q&A]
I wonder if Meta-AconC need much more GPU memory.

I am training on my custom dataset with a maximum batch-size of 22.

When I change the activation function from SiLU to Meta-AconC like this, I can't train with a batch-size larger than 10.
https://github.com/ultralytics/yolov5/blob/c9c95fb282322bc2928f44488f2467c5f4104f09/models/common.py#L34-L55

Is my method incorrect? Or maybe I overlooked something?

**Comment by glenn-jocher** [comment]
@zlj-ky yes naturally different activations require different resources

**Comment by zlj-ky** [comment]
@glenn-jocher Thanks for your reply! I think the increase of parameters brought by Meta-AconC is a little too much , compared with the performance improvement it brings. BTW, In the paper of acon, It suggests that  donot change all the activa-
tions to meta-ACON for deep CNN to avoid overfitting. In my opinion, it is also related to the increase of parameters. Therefore, I just wonder  if you have considered using meta-ACON selectively in YOLOV5 instead of replace all the SiLU.
![图片](https://user-images.githubusercontent.com/74093297/163578655-ccd2e823-47d8-4470-9f1c-344bd34c6d4e.png)

**Reply by glenn-jocher** [reply]
@zlj-ky no, I don't have any other tests except those performed in the YOLOv5 activations study.

**Comment by zhouyt11** [comment]
When I use this activation function, I train 50 epoch on the VisDrone dataset, but the map is only 0.002, in addition, map, p, r is 0 on the default coco dataset, what is the situation with this?
![image](https://user-images.githubusercontent.com/100337221/194024023-261cfc83-dfe8-42ac-a9b0-333339ddc36c.png)

**Reply by glenn-jocher** [reply]
@zhouyt11 see https://github.com/ultralytics/yolov5/issues/2891
